analytics.js

```diff
@@ -923,7 +923,9 @@ class Analytics {
       transformToServerNames(rudderElement.message.integrations);
 
       // Removing true values from clientSuppliedIntegrations except All
-      clientSuppliedIntegrations = Object.keys(clientSuppliedIntegrations)
+      const tempClientSuppliedIntegrations = Object.keys(
+        clientSuppliedIntegrations
+      )
         .filter(
           (integration) =>
             integration !== "All" &&
```

Comment:
We shouldn't be filtering the `All` entry.

Reply:
updated